pydis_site/templates/home/timeline.html

```diff
@@ -0,0 +1,449 @@
+                    <span class="cd-timeline__date">May 25th, 2020</span>
+                </div>
+            </div> <!-- cd-timeline__content -->
+        </div> <!-- cd-timeline__block -->
+
+        <div class="cd-timeline__block">
+            <!-- ... -->
+            <div class="cd-timeline__img cd-timeline__img--picture">
+                <img src={% static "images/timeline/cd-icon-picture.svg" %} alt="Picture">
+            </div> <!-- cd-timeline__img -->
+
+            <div class="cd-timeline__content text-component">
+                <h2>Python Discord is now listed on python.org/community</h2>
+                <p class="color-contrast-medium">lorem ipsum</p>
+
+                <div class="flex justify-between items-center">
+                    <span class="cd-timeline__date">May 28th, 2020</span>
+                </div>
+            </div> <!-- cd-timeline__content -->
+        </div> <!-- cd-timeline__block -->
+
+        <div class="cd-timeline__block">
+            <!-- ... -->
+            <div class="cd-timeline__img cd-timeline__img--picture">
+                <img src={% static "images/timeline/cd-icon-picture.svg" %} alt="Picture">
+            </div> <!-- cd-timeline__img -->
+
+            <div class="cd-timeline__content text-component">
+                <h2>Python Discord Public Statistics are now live</h2>
+                <p class="color-contrast-medium">lorem ipsum</p>
+
+                <div class="flex justify-between items-center">
+                    <span class="cd-timeline__date">Jun 4th, 2020</span>
+                </div>
+            </div> <!-- cd-timeline__content -->
+        </div> <!-- cd-timeline__block -->
+
+        <div class="cd-timeline__block">
+            <!-- ... -->
+            <div class="cd-timeline__img cd-timeline__img--picture">
+                <img src={% static "images/timeline/cd-icon-picture.svg" %} alt="Picture">
+            </div> <!-- cd-timeline__img -->
+
+            <div class="cd-timeline__content text-component">
+                <h2>PyDis summer code jam 2020 with the theme “Early Internet” and Django as the technology</h2>
+                <p class="color-contrast-medium">lorem ipsum</p>
+
+                <div class="flex justify-between items-center">
+                    <span class="cd-timeline__date">Jul 31st, 2020</span>
+                </div>
+            </div> <!-- cd-timeline__content -->
+        </div> <!-- cd-timeline__block -->
+
+        <div class="cd-timeline__block">
+            <!-- ... -->
+            <div class="cd-timeline__img cd-timeline__img--picture">
+                <img src={% static "images/timeline/cd-icon-picture.svg" %} alt="Picture">
+            </div> <!-- cd-timeline__img -->
+
+            <div class="cd-timeline__content text-component">
+                <h2>Python Discord is now the new home of the PyWeek event!</h2>
+                <p class="color-contrast-medium">lorem ipsum</p>
```

Comment:
```suggestion
                <p class="color-contrast-medium">PyWeek, a game jam that has been running since 2005, joins Python Discord as one of our official events. Find more information about PyWeek on <a href="https://pyweek.org/">their official website</a>.</p>
```